test/utils.js

```diff
@@ -17,7 +20,63 @@ const setupDB = async() => {
 	})
 }
 
+const fixtures = {
+	async insertUser(overrides = {}){
+		const userData = {
+			firstName:faker.name.firstName(),
+			lastName :faker.name.lastName(),
+			email : faker.internet.email(),
+			password : faker.internet.password(),
+			gender : faker.helpers.arrayElement(['male', 'female']),
+			role : faker.helpers.arrayElement(['admin', 'user']),
+			department : faker.helpers.arrayElement([
+				'marketting', 'finance', 'sales', 'technology']),
+			address : faker.address.city(),
+			jobRole : faker.random.word(),
+			refreshToken : faker.datatype.uuid()
+			
+		}
+		const newData = {...userData, ...overrides}
+		const passwordHash = await genPasswordHash(newData.password)
+		const newUser = await db.query(
+			`INSERT INTO users
+			("firstName", "lastName", email, "passwordHash",
+				 gender, role, department, address, "jobRole", "refreshToken")
+			 VALUES ($1 , $2 , $3 , $4, $5, $6, $7, $8, $9 , $10) RETURNING *
+			`, [newData.firstName , newData.lastName , newData.email, 
+				passwordHash, newData.gender, newData.role ,
+				newData.department, newData.address, newData.jobRole,
+				newData.refreshToken
+			]
+		)
+		return newUser.rows[0]
+		
+	},
+	async insertPost(user,overrides = {}){
```

Comment:
@beblicarl the overrides is just fine for this. all you have to do is pass it `userId`